src/client/pages/Assessment/Assessment.tsx

```diff
@@ -101,6 +104,15 @@ const Assessment: React.FC = () => {
               </SectionWrapper>
             }
           />
+          <Route
+            path={`${ClientRoutes.Assessment.Collaborator.Root.path.relative}/*`}
+            element={
+              <Routes>
+                <Route path={ClientRoutes.Assessment.Collaborator.User.path.relative} element={<User />} />
```

Comment:
Can element be directly User? `element={User}`

Reply:
no. I tried various options but at the moment this is the only one that works.